lib/apply-extends.js

```diff
@@ -16,7 +16,22 @@ function getPathToDefaultConfig (cwd, pathToExtend) {
   return path.resolve(cwd, pathToExtend)
 }
 
-function applyExtends (config, cwd) {
+function mergeDeep (target, config1, config2) {
+  const isObject = obj => obj && typeof obj === 'object' && !Array.isArray(obj)
+  for (let key of Object.keys(config1)) {
```

Comment:
I'd replace that first `for` loop by an `object.assign`

```suggestion
  Object.assign(target, config1)
```

Reply:
Fixed.